src/components/TodoList/TodoList.tsx

```diff
@@ -1,51 +1,95 @@
 import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[];
+  visible: boolean;
+  setTodoOnClick: (arg: Todo) => void;
+  showModal: (arg: boolean) => void;
```

Comment:
Consider renaming the `setTodoOnClick` and `showModal` props to more descriptive names like `onTodoSelect` and `onModalToggle` to improve readability and maintainability.